test/cypress/integration/users-management-ui.spec.js

```diff
@@ -0,0 +1,159 @@
+import {
+  TEST_ID_USER_LIST_TABLE,
+  TEST_ID_DETAIL_USER_TABLE,
+  TEST_ID_USER_SEARCH_FORM,
+  TEST_ID_USER_PROFILE_FORM,
+  TEST_ID_USER_LIST_PAGE,
+  TEST_ID_USER_FORM,
+  TEST_ID_USER_AUTHORITY_TABLE,
+  TEST_ID_USER_AUTHORITY_PAGE_FORM, TEST_ID_USER_AUTHORITY_MODAL,
+} from '../../../src/ui/test-const';
+
+describe('Users Management', () => {
+  const USERNAME = 'admin';
+
+  beforeEach(() => {
+    cy.appBuilderLogin();
+    cy.closeWizardAppTour();
+  });
+
+  afterEach(() => {
+    cy.appBuilderLogout();
+  });
+
+  describe('Validate UI', () => {
```

Comment:
I would move this "validate UI" part into the react-testing-library codebase, since it's not so related with the flow. @avdev4j what do you think?